### PR TITLE
Draft for the Community front page and images

### PR DESCRIPTION
**Pull request by apawlik**

### CODE REVIEW

**Comment by apawlik** [comment]
@nokome , @oliver---- , @michael Feedback appreciated on 
https://github.com/stencila/website/blob/community/src/community/index.html
Thanks!

File: src/community/index.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    <div class="column">
+      <section class="hero">
+        <div class="hero-body">
+          <div class="container">
+              <div class="section has-text-centered">
+                <h1 class="title is-2">Community</h1>
+                <p>
+                  Stencila is a community-owned and community-driven open source project. Stencila users and contributors are at the core
+                  of the software existence and its future. Community growth and engagement are our top priority. We are putting a lot of effort
+                  to create inclusive and friendly environment for everyone who wants to use and develop Stencila.
```
Comment (oliver----) [inline src/community/index.html]: should read 'create an inclusive...'

**Comment by nokome** [comment]
Looks good. I've made a few changes, including Oliver's comment. I'm going to merge into master. Any further comments/changes can always be made later.

**Comment by apawlik** [comment]
Thanks!

**Pull request merged**